sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/GenerateSchemaTest.kt

```diff
@@ -38,6 +38,12 @@ class GenerateSchemaTest {
     // verify
     assertThat(schemaFile.exists())
         .isTrue()
+    val lastModified = schemaFile.lastModified()
+
+    while (System.currentTimeMillis() - lastModified <= 1000) {
+      // last modified only updates per second.
```

Comment:
this is such a dumb piece of knowledge that is now in my brain but i doubt will ever be useful again

also PTAL

Reply:
Amazing. You could also `Thread.sleep(2_000)` or something but it doesn't matter